apps/server/src/modules/h5p-editor/temporary-file-storage/temporary-file.entity.ts

```diff
@@ -0,0 +1,33 @@
+import { Entity, Property } from '@mikro-orm/core';
+import { ITemporaryFile, IFileStats } from '@lumieducation/h5p-server';
+import { BaseEntity } from '@shared/domain';
+
+@Entity({ tableName: 'h5p-editor-temp-file' })
+export class TemporaryFile extends BaseEntity implements ITemporaryFile, IFileStats {
+	/**
+	 * The name by which the file can be identified; can be a path including subdirectories (e.g. 'images/xyz.png')
+	 */
+	@Property()
+	filename: string;
+
+	@Property()
+	expiresAt: Date;
+
+	@Property()
+	ownedByUserId: string;
+
+	@Property()
+	birthtime: Date;
```

Comment:
Does this mean the same as "createdAt" in other entities? Then we should stick with that name.

Reply:
Haven't seen that it comes from the interface. Then nevermind...